### PR TITLE
Rewrote and formatted Readme

### PR DESCRIPTION
**Pull request by jodahoney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,33 @@
-# 3x-Plus-1
+# 3x-Plus-1 #
 
-3x-Plus-1 is a lagorithm of encryption which uses the famous 3x+1 maths problem for solving the encryption problem. It is used the find the highest no where the 3x+1 calculator can reach and at which points based on the no of characters. 
 
-After finding the highest Possible no and no of steps to reach the no it can be used in two ways, either be used as salt with sha256 Password or can also be used as sha56( sha56(password) + sha56(highestPossible) + sha56(highestPoint) ).
+`3x-Plus-1` is a logarithmic encryption scheme that is based on the Collatz Problem.
```
Comment (rishabh-live) [inline README.md]: Change logarithmic to algorithm

**Review by rishabh-live — CHANGES_REQUESTED** [review]
Every Thing Looks Great just change logarithmic to algorithm on first line

**Review by rishabh-live — COMMENTED** [review]
Perfect

**Comment by rishabh-live** [comment]
@jodahoney You Have closed this I cannot merge you won't get contributions in hacktoberfest

**Comment by jodahoney** [comment]
@rishabh-live Sorry, did not mean to close.

**Review by rishabh-live — APPROVED** [review]

**Pull request merged**